### PR TITLE
Add October 2022 newsletter

### PR DESCRIPTION
**Pull request by CollierCZ**
- Add October 2022 newsletter
- Add newsletter articles to topics list

----
:books: Documentation preview :books:: https://writethedocs-www--1824.org.readthedocs.build/en/1824/

### CODE REVIEW

File: docs/blog/newsletter-october-2022.rst [hunk 1]
```diff
@@ -0,0 +1,109 @@
+.. post:: October 05, 2022
+   :tags: newsletter
+
+########################################
+Write the Docs Newsletter – October 2022
+########################################
+
+Helo once again to the wonderful Write the Docs community. While the weather may be getting cooler in my part of the world, I can always find something to warm my heart in this community, if not necessarily the rest of my body.
```
Comment (plaindocs) [inline docs/blog/newsletter-october-2022.rst]:
```suggestion
Hello once again to the wonderful Write the Docs community. While the weather may be getting cooler in my part of the world, I can always find something to warm my heart in this community, if not necessarily the rest of my body.
```

Comment (plaindocs) [reply docs/blog/newsletter-october-2022.rst]: I'd consider rewording this paragraph, it feels extra verbose and leans into the cold/warm thing kinda hard.

File: docs/blog/newsletter-october-2022.rst [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+Some of the best of the community was on display last month at the Prague conference. If you weren't able to see the talks as they were broadcast and join in on the productive conversations, you can at least catch up on `videos of all talks <https://youtube.com/playlist?list=PLZAeFn6dfHpm1PRgp84X5jh9Jca_KTJSF>`__, including the lightning talks. And if you want a visual guide to each talk, you can find `sketchnotes for each talk <https://www.flickr.com/photos/writethedocs/>`__ on Flickr. The talks represent a variety of perspectives and experiences so you'll probably find something to inspire you.
```
Comment (plaindocs) [inline docs/blog/newsletter-october-2022.rst]: > Some of the best of the community

people? work? ideas?

File: docs/blog/newsletter-october-2022.rst [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+* (3): Know most of what needs to be done.
+* (5): Some things still need to be figured out, but the pieces are all there. No major unknowns.
+* (8): Lot of things to figure out. Some unknowns.
+* (???): Work that you can’t estimate, like research (it could last forever!). These usually need either breaking down or time-boxing.
+
+-------------------
+Gerunds in headings
+-------------------
+
+Gerunds, words formed with verbs but functioning as nouns like playing, dancing, and eating, have been considered a problem in headings in technical documentation. Arguments for resistance include the fact that gerunds introduce a lot of visual repetition, which hinders skimming and causes noise. Gerunds are also inconsistent with `how people browse the internet <https://idratherbewriting.com/2008/02/11/are-gerunds-in-topic-titles-problematic-in-search-results/>`__. For example, people are more likely to search for "remove user account" than "removing user accounts". Moreover, gerunds can be challenging to translate and localize since some languages do not have identical counterparts.
```
Comment (plaindocs) [inline docs/blog/newsletter-october-2022.rst]: Would this read better with Docs relevant gerunds, instead of `playing and dancing`?

Comment (plaindocs) [reply docs/blog/newsletter-october-2022.rst]: Should they be italicized?

**Review by plaindocs — APPROVED** [review]
Solid stuff! Love the added topics links!

File: docs/blog/newsletter-october-2022.rst [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+- `Remote Python API and Developer Documentation Technical Writing Consultant  <https://jobs.writethedocs.org/job/999/remote-python-api-and-developer-documentation-technical-writing-consultant/>`__, Good Words LLC (Remote)
+- `Technical Writer (Kotlin) <https://jobs.writethedocs.org/job/972/technical-writer-kotlin/>`__, JetBrains (Remote or various locations)
+- `Technical Writer <https://jobs.writethedocs.org/job/969/technical-writer/>`__, Adyen (Amsterdam, Netherlands)
+- `Technical Writer <https://jobs.writethedocs.org/job/969/technical-writer/>`__, Dolby Poland Sp. z o.o. (Remote [Poland])
+- `Technical Writer (PhpStorm) <https://jobs.writethedocs.org/job/971/technical-writer-phpstorm/>`__, JetBrains (Remote or various locations)
+- `Product Designer - Documentation <https://jobs.writethedocs.org/job/977/product-designer-documentation/>`__, Adyen (Amsterdam, Netherlands)
+- `Internal Developer Advocate | Documentation <https://jobs.writethedocs.org/job/976/internal-developer-advocate-documentation/>`__, Adyen (Amsterdam, Netherlands)
+
+*To apply for this job and more, visit the* `Write the Docs job board <https://jobs.writethedocs.org/>`_.
```
Comment (hillaryfraley) [inline docs/blog/newsletter-october-2022.rst]:
```suggestion
*To apply for these jobs and more, visit the* `Write the Docs job board <https://jobs.writethedocs.org/>`_.
```

**Review by hillaryfraley — APPROVED** [review]
Just one suggestion!

File: docs/blog/newsletter-october-2022.rst (hunk 3 above)
Comment (plaindocs) [reply docs/blog/newsletter-october-2022.rst]: Super minor nit which you should probably ignore, by Docs relevant I meant more thing that you'd write about: Installing Configuring, Running

Comment (CollierCZ) [reply docs/blog/newsletter-october-2022.rst]: Yeah, those were what I was aiming for, but then got distracted halfway through and couldn't think of examples. :facepalm: `creating` came up in the discussions, but not the other two.

**Pull request merged**